cfn/paws-collector.template

```diff
@@ -85,6 +90,14 @@
             "AllowedPattern" : "(?:^\\d{4}(-\\d{2}){2}T(\\d{2}:){2}\\d{2}Z$)?"
         }
     },
+    "Conditions" : {
+        "CreateCredentialsBucket" : {"Fn::And": [
```

Comment:
S3 bucket may exist already

Reply:
thats a really good point. i dont really know a good way of generating a good unique name.